.gitignore

```diff
@@ -7,6 +7,8 @@
 # Ignore bundler config.
 /.bundle
 
+rspec.txt
+
```

Comment:
Not sure why we need this?

Reply:
Looks like it's the default --out example when writing the test run to an output file

Reply:
Oki